core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -112,6 +112,7 @@ class ReplicaFetcherThread(name: String,
         trace(s"Follower ${replica.brokerId} set replica high watermark for partition $topicPartition to $followerHighWatermark")
       if (quota.isThrottled(topicPartition))
         quota.record(records.sizeInBytes)
+      BrokerTopicStats.getBrokerAllTopicsStats.replicationBytesInRate.mark(records.sizeInBytes)
```

Comment:
For `bytesOut`, we track everything in the same place (`KafkaApis`).

For `bytesIn`, it's a bit scattered. For the leader we do the check in `ReplicaManager`, which means that we count the bytes from `GroupCoordinator.doSyncGroup`, but we don't count the bytes from `GroupMetadataManager.cleanupGroupMetadata`. Not sure if this is by design. @junrao probably knows.